src/popup/modules/QrCreator.js

```diff
@@ -88,8 +111,8 @@ export function setSize(size) {
  * @returns {void}
  */
 export function setText(text) {
-    setTextInternal(text);
-    UserInterface.setQrInputFieldText(text);
+    setTextInternal(preprocess(text));
```

Comment:
I know a totally minor "optimisation", but please avoid redoing the same step (pre-processing) again and again. Do it once and use the variable for both.